OpTestLogger.py

```diff
@@ -0,0 +1,102 @@
+
```

Comment:
Can you add license header content here?

Reply:
Done